package.json

```
@@ -3,9 +3,8 @@
 	"name": "proposal-pattern-matching",
 	"description": "A repository for ECMAScript pattern matching proposals.",
 	"scripts": {
-		"start": "npm run build-loose -- --watch",
-		"build": "npm run build-loose -- --strict",
-		"build-loose": "ecmarkup --verbose spec.emu index.html"
+		"start": "ecmarkup spec.emu index.html --watch",
```

```suggestion
		"start": "npm run build -- --watch",
```
to avoid repeating the ecmarkup command